starknet_py/tests/e2e/client/conftest.py

```diff
@@ -20,15 +20,17 @@
     prepare_net_for_tests,
     PreparedNetworkData,
 )
-from starknet_py.tests.e2e.conftest import contracts_dir
+from starknet_py.tests.e2e.conftest import contracts_compiled_dir
 
 directory = os.path.dirname(__file__)
 
 
 async def prepare_network(
     new_gateway_account_client: AccountClient,
 ) -> PreparedNetworkData:
-    contract_compiled = Path(contracts_dir / "balance_compiled.json").read_text("utf-8")
+    contract_compiled = Path(
+        contracts_compiled_dir / "balance_compiled.json"
+    ).read_text("utf-8")
```

Comment:
If we change all `contract_source` to `contract_compiled` we should save some time. Tests will be faster 🔥

Reply:
Of course not in tests testing contract compilation 😅